tests/drum/test_unstructured_mode.py

```diff
@@ -264,14 +265,17 @@ def test_response_one_var_return(
                                 assert response.content == data_text.encode(UTF8)
 
                 # sending binary data
-                headers = {"Content-Type": "application/octet-stream;"}
-                response = requests.post(url=url, data=data_bytes, params=params, headers=headers)
-                assert response.ok
-                content_type_header = response.headers["Content-Type"]
-                mimetype, content_type_params_dict = werkzeug.http.parse_options_header(
-                    content_type_header
-                )
-                assert "application/octet-stream" == mimetype
-                # check params dict is empty
-                assert not any(content_type_params_dict)
-                assert response.content == data_bytes
+                for ct in ["application/octet-stream;", "application/x-www-urlencoded;"]:
```

Comment:
with this we have 2 tests in 1 -- let's move it to `pytest.mark.parametrize`

Reply:
But parametrize will run test two times, which will start/stop server two times so overall runtime will be longer.
Of course it's negligible for a single test, but we have a lot of such cases, which I implemented without parameterized to save a lot of time.